src/backend/gl/src/state.rs

```diff
@@ -42,16 +41,17 @@ pub fn bind_rasterizer(gl: &gl::Gl, r: &pso::Rasterizer, is_embedded: bool) {
     };
 
     match r.cull_face {
-        Some(face) => unsafe {
+        Some(cf) if !cf.is_empty() => unsafe {
             gl.Enable(gl::CULL_FACE);
-            gl.CullFace(match face {
-                Front => gl::FRONT,
-                Back => gl::BACK,
+            gl.CullFace(if cf.contains(pso::Face::FRONT) {
```

Comment:
same here - needs to use `gl::FRONT_ANT_BACK` for all faces

Reply:
🐜 A new kind of GL face? 😆